todo-app/src/Components/AddTodoItem.js

```diff
@@ -0,0 +1,99 @@
+//jshint esnext: true
+
+import React, { Component } from 'react'
+
+import Button from './Button';
+import InputField from './InputField';
+import './styles/addTodoItem.css';
+
+export default class AddTodoItem extends Component {
+
+    state = {
+        adding: false,
```

Comment:
It's a little picky, but I might change this to something like `active`, as the component seems to have two states: active and inactive.